cmake/platform.cmake

```diff
@@ -107,6 +107,12 @@ elseif(UNIX OR MINGW)
              if (CMAKE_SYSTEM_PROCESSOR STREQUAL CMAKE_HOST_SYSTEM_PROCESSOR)
```

Comment:
(random spot)

Could you please extend [`src/cpu/platform.hpp`](https://github.com/oneapi-src/oneDNN/blob/master/src/cpu/platform.hpp) and [`src/cpu/README.md`](https://github.com/oneapi-src/oneDNN/blob/master/src/cpu/README.md) with the corresponding macro `DNNL_POWER9`?

Reply:
The actual macro name should be aligned with Cmake one. Thus, probably we first need to address https://github.com/oneapi-src/oneDNN/pull/767#discussion_r447102653.

Reply:
Sure, I’m on it.

Reply:
Thanks!